config/initializers/string_extension.rb

```diff
@@ -2,4 +2,19 @@
   def to_hex
     Digest::MD5.hexdigest("z#{self}").to_s[0..5].upcase
   end
+
+  # Public: Converts a string to a boolean.
+  #   Handles various boolean synonyms as well (eg. 't' or 'true' or 'yes', etc...)
+  #
+  # Examples
+  #
+  #   "yes".to_bool
+  #   # => true
+  #
+  # Returns the converted boolean.
+  def to_bool
```

Comment:
i see... how would you suggest we handle an argument like "neutral".to_bool ? default to nil?